src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -66,7 +66,7 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
 
         <div class="col-md-4 col-sm-5 col-xs-5 wi-state-btn pull-right padding0">
           <div dropdown class="dropdown card-pf-time-frame-filter detail-status-dropdown">
-            <button dropdown-open 
+            <button dropdown-open [disabled]='addNewWI'
```

Comment:
@joshuawilson yeah, so in case of adding, no API should be fired until the title is valid. So, she updated the code and it looks fine now.